### ISSUE TITLE
command-line options not visible when using PowerShell with Solarized Dark color scheme

### ISSUE THREAD

**Issue by imwithsam** [microsoft/terminal#6716]
# Environment

```none
Windows build number: 10.0.19041.0
Windows Terminal version: 1.0.1401.0

Any other software? PowerShell 7.0.2 and 5.1.19041.1
```

# Steps to reproduce

1. Set the Windows Terminal color scheme to 'Solarized Dark'
2. Bring up PowerShell in Windows Terminal
3. Type any command with an option (e.g. `git help -a`)

# Expected behavior

You can read the entire command line (e.g. `git help -a`)

# Actual behavior

You cannot read the command-line option (e.g. `git help` is visible but `-a` is not)

# Notes

As far as I can tell, this issue appears to be specific to Windows Terminal with PowerShell and the Solarized Dark color scheme.

**Comment by Phistr90** [comment]
I have the same problem

**Comment by abjklk** [comment]
The issue is with the present `colorScheme `values as evident in the `defaults.json` file.
The attributes `brightBlack `and `background` have the same hex value of **`#002B36`**

A temporary fix would be to create your own custom Solarized Dark color scheme.
Follow this documentation to create a custom theme: [https://docs.microsoft.com/en-us/windows/terminal/customize-settings/color-schemes](url)

**Comment by zadjii-msft** [comment]
Thanks for the suggestion! Please make sure to [search for duplicates](https://github.com/microsoft/terminal/issues?q=is%3Aissue+%22solarized%22+is%3Aclosed) when filing new issues. This has been reported a bunch of times already - I think we all regret shipping Solarized as a default theme.

/dup #6696 #6424 #6390 #6286 #6283 #6208 #6121 #6061 #5896, and a ton more. Resolved by #6617 #6618 #6489 (all three are currently in review)

**Comment by imwithsam** [comment]
I searched for duplicates but only in Open issues. Thank you for the update!
